src/site/content/en/blog/access-devices/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Accessing hardware devices on the web
+subhead: |
+  Pick the appropriate API to communicate with a hardware device of your choice.
+description: |
+  This article helps Web developers pick the right hardware API based on a given device.
+authors:
+  - beaufortfrancois
+date: 2021-01-19
+updated: 2021-01-19
+hero: hero.jpg
+thumbnail: thumbnail.jpg
+alt: A woman sitting in front of a wooden desk photo.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - devices
+---
+
+My goal is to help you, web developer, pick the appropriate API to communicate
+with a hardware device of your choice (e.g., a webcam, a microphone, a mouse, a
+keyboard, a stylus, an NFC tag, a Bluetooth device, or an exotic hardware
+device).
+
+There are many APIs to choose from. By answering a few questions on this device, you
+should find out the API to use and have some documentation to get started
+accessing this device.
+
+## 1. Do you want to access video stream from this device? {: #video }
+
+If so, use [`MediaDevices.getUserMedia()`] to get a live video stream from this
+device and learn about [capturing video]. You can also [control the camera's
+pan, tilt, and zoom], and other camera settings such as [brightness and
+contrast], and even [take still images].
+
+## 2. Do you want to access audio streams from this device? {: #audio }
+
+If so, use [`MediaDevices.getUserMedia()`] to get a live audio stream from this
+device and learn about [capturing audio]. You can also use [Web Audio] to add
+effects to audio, create audio visualizations, or apply spatial effects (such as
+panning). Check out [how to profile the performance of Web Audio apps] in Chrome
+as well.
+
+## 3. Do you want to receive input events from this device? {: #input }
+
+If so, try listening for [Keyboard] and [Pointer] events. If this device is a
+game controller, use the [Gamepad] API to know which buttons are being pressed
+and which axes moved.
+
+## 4. Is this device an NFC tag? {: #nfc }
+
+Then, try [Web NFC] to read and write to this device when it's in close
+proximity to the user's device (usually 5-10 cm, 2-4 inches).
+
```

Comment:
A few other types of devices to include here are storage devices (File System Access API), printers (window.print()), network adapters (fetch()) and authenticators (WebAuthn).

Reply:
I didn't know about network adapters ;)